lib/core/src/display/shape/primitive/glsl/shape.glsl

```diff
@@ -112,29 +112,70 @@ Id new_id_layer (BoundSdf sdf, int i) {
 }
 
 
+// Premultiplied
+struct Color {
```

Comment:
I wonder if "nice docs rule" apply also to glsl code? If yes, then this is certainly not nice doc.

You could also add docs to the other structures, although they can be also treated as "private API"

Reply:
For now its private API. But I want to make docs nicer there in general